dynawo/sources/Models/CPP/ModelNetwork/DYNModelNetwork.cpp

```diff
@@ -443,7 +443,10 @@ ModelNetwork::initializeFromData(const shared_ptr<DataInterface>& data) {
     data->setReference("p2", id, id, "P2_value");
     data->setReference("q2", id, id, "Q2_value");
     data->setReference("state", id, id, "state_value");
-    data->setReference("tapIndex", id, id, "step_value");
+    if (modelTwoWindingsTransformer->getModelRatioTapChanger() || modelTwoWindingsTransformer->getModelPhaseTapChanger())
```

Comment:
Why 2 if and not a or?

Reply:
no reason I just added the else if after a first version so I did not harmonized, I can do only one if